tools/clang/unittests/HLSL/FileCheckerTest.cpp

```diff
@@ -722,6 +722,9 @@ class FileRunTestResultImpl : public FileRunTestResult {
     // Iterate over all RUN lines
     for (auto &cmd : cmds) {
       RunFileCheckFromCommands(cmd.c_str(), fileName);
+      // If any of the RUN cmd fails then skip executing remaining cmds
+      // and report the error
```

Comment:
This fixes a bug in FileChecker when running multiple RUN lines. Upon encountering a failure with a RUN line, it should stop executing the remaining RUN lines, otherwise result of a subsequent RUN line could overwrite/mask the previous failure.